### PR TITLE
[NEW] Default servers improvements

### LINKED ISSUE #455: servers.json and /allusers setup
Hi,

I'd like to deploy a /allusers setup on Windows. Is it possible to place a servers.json file as a kind of template globally, which is copied then to the user's folder?

"The servers.json file needs to be placed in the %APPDATA% folder for the User not the System wide one."

This is a problem in a /allusers setup. How could I copy the  servers.json file to the %APPDATA% folder if the program is started first time.

Ciao
Marcus

### PR DESCRIPTION
**Pull request by gdelavald**
@RocketChat/core 

Closes #455
Based on Pull Request #471 from @localguru (Thank you so much for this)
Synced with develop to fix the CI build and did a small refactor on the code.

### CODE REVIEW

**Comment by gdelavald** [comment]
Seems like the `process.cwd()` can't be fully trusted when the app is packaged, to get the actual path Electron provides `getAppPath()` method on the API, as listed [here](https://github.com/electron/electron/blob/master/docs/api/app.md#appgetapppath).

**Comment by gdelavald** [comment]
I'll hang on to this until we find a solution that suits the comments on PR #312, based on installation time server configuration.

**Comment by gdelavald** [comment]
I'm happy with the current state of this PR. I tested in several use cases with both Mac and Windows versions, I need to do some testing for linux distros, but it should be working as intended afaik.

File: src/scripts/servers.js [hunk 1]
```diff
@@ -65,10 +65,13 @@ class Servers extends EventEmitter {
 
         // Load server info from server config file
         if (Object.keys(hosts).length === 0) {
-            const serverFileName = 'servers.json';
-            const userDataDir = jetpack.cwd(remote.app.getPath('userData'));
+            const pathToServerJson = jetpack.path(
```
Comment (gdelavald) [inline src/scripts/servers.js]: Now we check the user APPDATA folder first, then the system installation folder (for /allusers) and then the packaged file (if we do a branded package).

**Comment by localguru** [comment]
+1 THANKS!

**Pull request merged**